content/applications/hr/referrals.rst

```diff
@@ -0,0 +1,707 @@
+
+Click on the photo to level up the user. The user's avatar changes to the current level, and the
+ring around the photo is updated to indicate the current amount of points.
+
+.. image:: referrals/level-up.png
+   :align: center
+   :alt: A 'Click to level up!' appears beneath the user's image, and a large 'Level up!' appears
+         above their image.
+
+.. note::
+   Once a user has reached the highest configured level, they will continue to accrue points that
+   can be redeemed for rewards, but they are no longer able to level up. The ring around their photo
+   remains solid cyan.
+
+Alerts
+======
+
+In the Referrals application, it is possible to post a message, also referred to as an *alert*, at
+the top of the dashboard to share important information with users. These alerts appear as a thin
+semi-transparent banner, with the word `New!` appearing on the far left. The text for the alert is
+in the center of the banner, and on the far right side is an :guilabel:`X`.
+
+Alerts appear on the main dashboard for the specified time configured on the individual alert. If a
+user does not wish to see a specific alert again, click the :guilabel:`X` in the far right side of
+the alert. This removes the alert from the dashboard and will not appear again, even when opening
+the Referrals application for the first time in a new session.
+
+.. image:: referrals/alerts.png
+   :align: center
+   :alt: Two alert banners appear above the user's photo.
+
+Create an alert
+---------------
+
+To add a new alert, navigate to the :menuselection:`Referrals application --> Configuration -->
+Alerts`.
+
+Click :guilabel:`Create` and a blank alert form loads. Enter the following information on the form:
+
+- :guilabel:`Date From`: The date the alert starts. On this date, the alert will be visible on the
+  dashboard.
+- :guilabel:`Date To`: The date the alert ends. After this date, the alert will be hidden from view.
+- :guilabel:`Company`: The current company populates this field by default. To modify the company
+  the alert should be displayed for, select the company from the drop-down menu.
+- :guilabel:`Alert`: Enter the text for the alert. This message appears inside the alert banner on
+  the main dashboard
+- :guilabel:`On Click`: There are three options for the alert. Click the radio button next to the
+  desired selection. The options are:
+
+  - :guilabel:`Not Clickable`: The alert only displays text, there is no link to click.
+  - :guilabel:`Go to All Jobs`: The alert contains a link that when clicked, navigates to the
+    website with all the currently posted job positions.
+  - :guilabel:`Specify URL`: The alert contains a link to a specific URL, that when clicked,
+    navigates to that URL. When selected, a :guilabel:`URL` field appears below the :guilabel:`On
+    Click` section. Enter the URL in the field.
+
+.. image:: referrals/alert-form.png
+   :align: center
+   :alt: An alert form completely filled in with all selections entered.
```

Comment:
maybe good to annotate with a red box around the entire form? It blends in when rendered

Reply:
I went without the red box since I would have to highlight everything- and since there is a gray box surrounding the image, i think it's ok.